pkg/cli/importimage/importimage.go

```diff
@@ -39,7 +39,17 @@ var (
 		see the 'tag' command.`)
 
 	importImageExample = templates.Examples(`
+		# Update imported data for an already existing image stream.
 		oc import-image mystream
```

Comment:
If I recall correctly, it'll update only mystream:latest.

Reply:
Yes, you are right. Let me add some info about it as well.

Reply:
I have adjusted, PTAL.